src/components/Filter/Filter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Filter({ continent, setContinent }) {
+  return (
+    <div>
+      <select className='filters' value={continent} onChange={(e) => { setContinent(e.target.value);
+      }}>
+        <option value="All">All</option>
```

Comment:
I noticed a bug while using the app where nothing displays if I change the dropdown and then later change it back to "All". It looks like you're checking for `continent === 'all'` in your hook, but this dropdown here is setting the state to `'All'`. An easy solution would be to simply change the value here `value="all"`.